### ISSUE TITLE
GoogleColab notebook with genome input

### ISSUE THREAD

**Issue by xvazquezc** [soedinglab/spacepharer#7]
Hi there,

I was trying to run a single phage genome through the GoogleColab notebook and it always fails at the Krona step. The input is a single genome in a single contig and I select the "Phage genome" option. It runs without issues with the CRISPR spacers test input though.

This is one of the genomes I've tested: [Mendera.zip](https://github.com/soedinglab/spacepharer/files/7248103/Mendera.zip). And this is the error I get:

```
FileNotFoundError                         Traceback (most recent call last)

<ipython-input-4-7889d783dfc8> in <module>()
      4 from pathlib import Path
      5 suffix = "_lca.html" if input_type == 1 else "_lca_per_target.html"
----> 6 html = '<script>onload()</script></html>'.join(Path(jobname + suffix).read_text().rsplit('</html>', 1))
      7 display(HTML(html))

2 frames

/usr/lib/python3.7/pathlib.py in read_text(self, encoding, errors)
   1219         Open the file in text mode, read it, and close the file.
   1220         """
-> 1221         with self.open(mode='r', encoding=encoding, errors=errors) as f:
   1222             return f.read()
   1223 

/usr/lib/python3.7/pathlib.py in open(self, mode, buffering, encoding, errors, newline)
   1206             self._raise_closed()
   1207         return io.open(self, mode, buffering, encoding, errors, newline,
-> 1208                        opener=self._opener)
   1209
   1210     def read_bytes(self):

/usr/lib/python3.7/pathlib.py in _opener(self, name, flags, mode)
   1061     def _opener(self, name, flags, mode=0o666):
   1062         # A stub for the opener argument to built-in open()
-> 1063         return self._accessor.open(self, flags, mode)
   1064 
   1065     def _raw_open(self, flags, mode=0o777):

FileNotFoundError: [Errno 2] No such file or directory: 'test_lca_per_target.html'
```

**Comment by RuoshiZhang** [comment]
Hi,
We just fixed this error with 79d743022f7d1414a24349a4f029d542e7f4ecce, which is arising from creating db when a phage genome is given. Now it should work.

**Comment by xvazquezc** [comment]
Thanks, it works fine now.
